operations/messages.md

```diff
@@ -0,0 +1,145 @@
+# Messages
+
+## Get all messages
+
+Returns all messages from the [Message thread](messagethreads.md#message-thread) that was created by the client application.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/messages/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "MessageThreadIds": [
+        "c1478ff9-80b7-4ea2-94fc-ae4e009e1463"
+    ],
+    "CreatedUtc": {
+        "StartUtc": "2022-03-03T00:00:00Z",
+        "EndUtc": "2022-03-14T00:00:00Z"
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `MessageThreadIds` | array of string | required, max 1000 items | Unique identifiers of [Message threads](#message-thread) from where to return messages. |

```

Comment:
This link is not showing up in the preview, because '#message-thread' should be 'messagethreads.md#message-thread'